docs/contributor/README.md

```diff
@@ -1 +1,64 @@
-In this folder, you can add any developer-related documentation, for example, advanced installation options, testing strategy, governance, etc.
\ No newline at end of file
+# Architecture
+
```

Comment:
How about adding a short overview along with the diagram? Just 1-3 introductory sentences will do.

Reply:
fixed